osdk/src/commands/build/bin.rs

```diff
@@ -0,0 +1,170 @@
+    aster_elf: &AsterBin,
+    protocol: &BootProtocol,
+) -> AsterBin {
+    let target_name = get_current_crate_info().name;
+    let image_type = match protocol {
+        BootProtocol::LinuxLegacy32 => BzImageType::Legacy32,
+        BootProtocol::LinuxEfiHandover64 => BzImageType::Efi64,
+        _ => unreachable!(),
+    };
+    let setup_bin = {
+        let setup_install_dir = target_dir.as_ref();
+        let setup_target_dir = &target_dir.as_ref().join("linux-bzimage-setup");
+        match image_type {
+            BzImageType::Legacy32 => {
+                let target_json = legacy32_rust_target_json();
+                let gen_target_json_path = target_dir.as_ref().join("x86_64-i386_pm-none.json");
+                std::fs::write(&gen_target_json_path, target_json).unwrap();
+                let arch = SetupInstallArch::Other(gen_target_json_path.canonicalize().unwrap());
+                install_setup_with_arch(setup_install_dir, setup_target_dir, &arch);
+            }
+            BzImageType::Efi64 => {
+                install_setup_with_arch(
+                    setup_install_dir,
+                    setup_target_dir,
+                    &SetupInstallArch::X86_64,
+                );
+            }
+        };
+        setup_install_dir.join("bin").join("linux-bzimage-setup")
+    };
+    // Make the `bzImage`-compatible kernel image and place it in the boot directory.
+    let install_path = install_dir.as_ref().join(target_name);
+    info!("Building bzImage");
+    println!("install_path: {:?}", install_path);
+    make_bzimage(&install_path, image_type, &aster_elf.path, &setup_bin);
+
+    AsterBin {
+        path: install_path,
+        typ: AsterBinType::BzImage(AsterBzImageMeta {
+            support_legacy32_boot: matches!(protocol, BootProtocol::LinuxLegacy32),
+            support_efi_boot: false,
+            support_efi_handover: matches!(protocol, BootProtocol::LinuxEfiHandover64),
+        }),
+        version: aster_elf.version.clone(),
+        sha256sum: "TODO".to_string(),
+        stripped: aster_elf.stripped,
+    }
+}
+
+pub fn strip_elf_for_qemu(install_dir: impl AsRef<Path>, elf: &AsterBin) -> AsterBin {
+    let stripped_elf_path = {
+        let elf_name = elf.path.file_name().unwrap().to_str().unwrap().to_string();
+        install_dir.as_ref().join(elf_name + ".stripped.elf")
+    };
+
+    // We use rust-strip to reduce the kernel image size.
+    let status = Command::new("rust-strip")
+        .arg(&elf.path)
+        .arg("-o")
+        .arg(stripped_elf_path.as_os_str())
+        .status();
+
+    match status {
+        Ok(status) => {
+            if !status.success() {
+                panic!("Failed to strip kernel elf.");
+            }
+        }
+        Err(err) => match err.kind() {
+            std::io::ErrorKind::NotFound => panic!(
+                "`rust-strip` command not found. Please 
+                try `cargo install cargo-binutils` and then rerun."
```

Comment:
How about adding `cargo install cargo-binutils` to the README.md, `Requirements` section?